presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -696,6 +696,15 @@ public void testArrayMin()
         assertDecimalFunction("ARRAY_MIN(ARRAY [2.22222222222222222, 2.3])", decimal("2.22222222222222222"));
     }
 
+    @Test
+    public void testArrayMinWithNulls()
+    {
+        assertFunction("ARRAY_MIN(ARRAY [ARRAY[1, NULL], ARRAY[2, NULL]])", new ArrayType(INTEGER), Lists.newArrayList(1, null));
```

Comment:
These should be four separate test methods so they can pass or fail independently.